src/config.js

```diff
@@ -45,6 +45,7 @@ class Config {
     this.enabled = String(enabled) === 'true'
     this.debug = String(debug) === 'true'
     this.logInjection = String(logInjection) === 'true'
+    this.logInjectionTags = new Set(coalesce(options.logInjectionTags, ['service.name']))
```

Comment:
Set is available in node 8 which is the lowest we use but is there any other platform that this might not work with? I can't think of any but still wanted to bring up.

Reply:
The Set methods used should be available from node `0.12.18` (https://node.green/#ES2015-built-ins-Set) and it's nodejs tracing. I think it should be okay :slightly_smiling_face: